packages/core/bootstrap/src/lib/rate-limit/config.ts

```diff
@@ -30,19 +31,27 @@ export function get(context: AdapterContext): Config {
       logger.error(e.message)
     }
   }
-  let burstCapacity = 0
+  let burstCapacity1s = 0
+  let burstCapacity1m = 0
   if (enabled) {
     const provider = getEnv('RATE_LIMIT_API_PROVIDER') || context.name?.toLowerCase() || ''
     const tier = getEnv('RATE_LIMIT_API_TIER') || ''
     try {
-      const providerConfig = getBurstLimit(provider, tier)
-      burstCapacity = Number(providerConfig)
+      const limit = getHTTPLimit(provider, tier, 'rateLimit1s')
+      burstCapacity1s = Number(limit)
+    } catch {
+      // Ignore
```

Comment:
Should we at least add some logs here for debugging?

Reply:
The functions in their give their own more granular logs.